### DISCUSSION TITLE
Spacy 3.o NER Help

### DISCUSSION THREAD

**Discussion by srikar-kodati** [explosion/spaCy#7971] [Help: Coding & Implementations] [answered]
Hey all!

So I have a project where I am using spacy and trying to create new NERs for some text. For the configuration file for CLI, do i just copy and paste it and save it as a file on the same directory? Do I save it as a python file? Please help. Thank You

**Comment by polm** [comment, marked as answer]
I recommend reading through the [quickstart](https://spacy.io/usage/training#quickstart) section of the docs. If you read that you'll see that you can save the config as `config.cfg` in the same directory and run `spacy init config` and `spacy train`, specifying the config path, to use it.

You can save the config file anywhere as long as you specify it on the command line. We often use a directory called `configs`. You might find the [example projects](https://github.com/explosion/projects/tree/v3/tutorials) a useful reference.

**Reply by syed1604** [reply]
python -m spacy init fill-config base_config.cfg config.cfg  . Where do I need to run this line I'm a newbie any help would be appreciated.

**Reply by polm** [reply]
@syed1604 Hey, sorry you're having trouble, but please don't comment on an unrelated issue, if you have questions open a new thread. 

You need to run the `python -m` command in the terminal. `base_config.cfg` and `config.cfg` are file names.
